docs/pages/application-access/reference.mdx

```diff
@@ -1,105 +1,15 @@
 ---
-title: Application Access Reference Documentation
-description: Configuration and CLI reference documentation for Teleport Application Access.
+title: Application Access Reference
+description: Configuration and CLI reference for Teleport Application Access.
 ---
 
-## Configuration
+# Application Access Reference
 
-(!docs/pages/includes/backup-warning.mdx!)
-
```

Comment:
I've refactored these into includes (as planned) so we can have these in the Section reference as well as the "global" reference.